docs/development.md

```diff
@@ -24,6 +24,32 @@ make build-darwin-amd64
 
 Tests can be run with `make test`.
 
+## Integration Tests with CINC Test Kitchen
```

Comment:
What is the purpose of the CINC stuff? I've not seen this before do we have a standard for using it elsewhere?

Reply:
Everytime. I swear I went through and checked :upside_down_face:

Reply:
CINC is an open source fork of Chef. Chef's license changed as is not suitable for us. I use CINC's Auditor (chef inspec) to validate post install state.

Reply:
I agree testing is worth while. I'm ok with this for now but my fears are:
- bloat in our repos (maybe we move this tooling to an internal repo that just runs these types of tests)
- Complex tooling that doesn't work everywhere. 

As an action item can you write up a proposal for using this or submit something to the standards repo if we want to move forward with this? I'm ok with Stanza being the poc repo for this but I would like some discussion before we put this heavy tooling in everywhere.

Reply:
Sounds good :+1: